src/ulfius.c

```diff
@@ -585,7 +585,7 @@ int mhd_iterate_post_data (void * coninfo_cls, enum MHD_ValueKind kind, const ch
   
   struct connection_info_struct * con_info = coninfo_cls;
   size_t cur_size = size;
-  char * data_dup = nstrndup(data, size); // Force value to end with a NULL character
```

Comment:
Actually, the `n*` functions are defined in `orcania` repository which is a mini project containing among others `nstrdup` and `nstrndup`.

This functions is created to avoid a crash when a `data` is `NULL` or `size` is 0.

Ulfius depends on Orcania to compile:
https://github.com/babelouest/orcania